security-questionnaire.md

```diff
@@ -0,0 +1,73 @@
+In terms of technical capabilities, the proposed API is a DOM-based API. In
+fact, it could be implemented with all existing DOM capabilities that the
+platform already provides - and there are existing JavaScript implementations of
+the algorithm we propose. We are proposing this API for the sake of increased
+security in the light of attacks that those implementations have had to face
+(e.g., DOM Clobbering and Mutated XSS). Given previous security issues, we
+believe that a sanitizer working in unisono with the browser’s parsing
```

Comment:
wording nitpick: Not really sure, but in English the form "in unison" seems to be preferred over "in unisono".